src/readBranches.ts

```diff
@@ -12,6 +12,11 @@ const GRAPHQL_QUERY = `query ($repo: String!, $owner: String!, $after: String) {
       edges {
         node {
           name
+          associatedPullRequests(first: 10, states: OPEN) {
```

Comment:
Would `first: 1` be sufficient here, since we only need to check for existence?

```suggestion
          associatedPullRequests(first: 1, states: OPEN) {
```